build/build.rb

```diff
@@ -89,6 +89,10 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
```

Comment:
@jvlcek Sure,..but then are you saying that all seven places in the script should be modified to use Dir.chdir

Also, in the comment below, you say to use FileUtils.cp,...but couldn't FileUtils.cd be used instead of Dir.chdir as stated above?

From a procedure standpoint, is this standard practice to change the whole script and update from a ruby perspective when adding a new line?